pkg/flow/internal/controller/loader.go

```diff
@@ -19,7 +20,7 @@ import (
 	"go.opentelemetry.io/otel/codes"
 	"go.opentelemetry.io/otel/trace"
 
-	_ "github.com/grafana/agent/pkg/flow/internal/testcomponents" // Include test components
+	_ "github.com/grafana/agent/pkg/util/testcomponents" // Include test components
```

Comment:
Why did we move this package? 🤔 It doesn't look like anything outside of pkg/flow needs it, right?

Reply:
That was because of the test in `component/module/file` to prevent circular dependency, but it's no longer needed after I've moved the test, so putting the tetcomponents back where they were.